servers/src/test/java/tachyon/underfs/UnderFileSystemCluster.java

```diff
@@ -105,7 +105,7 @@ public static UnderFileSystemCluster getUnderFilesystemCluster(String baseDir,
    */
   public static boolean readEOFReturnsNegative() {
     // TODO Should be dynamically determined - may need additional method on UnderFileSystem
-    return (null != sUfsClz) && !(sUfsClz.equals("tachyon.LocalUnderFileSystem"));
+    return null != sUfsClz && sUfsClz.equals("tachyon.underfs.hdfs.LocalMiniDFSCluster");
```

Comment:
Sure, the name is pretty much hardcoded at the moment. When that class is used it implicitly means that hdfs is being tested.